tests/package-smoke/index.html

```
@@ -0,0 +1,2 @@
+<div id="app"></div>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)